python/tvm/relay/qnn/op/qnn.py

```diff
@@ -92,6 +92,44 @@ def requantize(
     )
 
 
+def upward(data):
```

Comment:
Would it make sense to add these as an attribute to `relay.round`? For example we could do `relay.round(x, mode=upward)`. I think that would be a cleaner interface unless there's a reason for having them as separate operators.

Reply:
I found this logic enough tested by basic tests and removed these operations.

Reply:
"I think itd also be nice to add a comment here explaining why we're only doing this for llvm. That will help contributors decide which to apply to new hardware."
I moved this logic to the default reqconfig. I added a little comment there. Other cases can be handled by specified reqconfig or requantize parameters.